File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>


long long solution(int a, int b) {
    long long answer = 0;
    if(a == b){
        return a;
    }
    if(a < b){
        for(a ; a<=b ; a++){
            answer +=a;            
        }
        return answer;
    }
    if(a > b){
        for(b ; b <= a ; b++){
            answer +=b;
        }
        return answer;
    }
    
}

File: C#/프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.cs
public class Solution {
    public long solution(int a, int b) {
            long answer = 0;

            if (a < b)
            {
                int temp = a;
                a = b;
                b = temp;
            }
            for (int i = b; i <= a; i++)
                answer += i;
            return answer;
    }
}